baseapp/options.go

```diff
@@ -233,3 +233,14 @@ func (app *BaseApp) SetInterfaceRegistry(registry types.InterfaceRegistry) {
 	app.grpcQueryRouter.SetInterfaceRegistry(registry)
 	app.msgServiceRouter.SetInterfaceRegistry(registry)
 }
+
+// SetStreamingService is used to set a streaming service into the BaseApp hooks and load the listeners into the multistore
+func (app *BaseApp) SetStreamingService(s sdk.StreamingService) {
```

Comment:
Where is the `Stream()` method used by `BaseApp`? If it's not needed by `BaseApp` it shouldn't be on the interface

Reply:
Its not used by the `BaseApp` but it is used by the parent `App`